### ISSUE TITLE
Feature requests: double click to edit, and expand all

### ISSUE THREAD

**Issue by ffxsam** [paralect/robomongo#908]
Two things would make this client even better:

1. The ability to double-click a record to edit.
2. A button to expand all records.

Thanks!

**Comment by ffxsam** [comment]
PS: Collapsing all the documents after editing one is kind of annoying.

**Comment by adambuczynski** [comment]
I agree that double clicking on a field to edit the value would be a great time saver.
